frontend/Dockerfile

```diff
@@ -6,29 +6,27 @@ COPY . .
 # Install dependencies
 RUN yarn install --frozen-lockfile
 
-# Copy environment variables file if it doesn't exist
-RUN test -e .env || cp .env.example .env
```

Comment:
If we are removing this, then do we need to define defaults for each ARG

Reply:
yes we can remove that.